dot/state/block.go

```diff
@@ -621,7 +621,7 @@ func (bs *BlockState) HandleRuntimeChanges(newState *rtstorage.TrieState,
 
 	codeSubBlockHash := bs.baseState.LoadCodeSubstitutedBlockHash()
 
-	if !codeSubBlockHash.Equal(common.Hash{}) {
+	if codeSubBlockHash != common.EmptyHash() {
```

Comment:
nit keep it to

```suggestion
	if codeSubBlockHash != common.Hash{} {
```

not sure why we have `EmptyHash` in the first place, might be worth removing it as well.

Reply:
```
dot/state/block.go:624:39: expected ';', found '{' (typecheck)
        if codeSubBlockHash != common.Hash{} {
```

Reply:
if I use 
```
codeSubBlockHash != (common.Hash{})
``` 
then error goes away, golang is so strange.